### ISSUE TITLE
NVDA loses track when webpage updates

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3336]
**Reported by parham on 2013-07-06 04:28**
The game [http://www.domainofheroes.com](Domain of Heroes) has been written by ASP.NET, but unlike most other .net apps, is accessible using Firefox. However, the game has multiple tabs (Information and chat, toon, skills, etc). When you are on one of these tabs (E.G. skills) and another tab is updated (E.G. information and chat), NVDA does not read anything when I try to read the contents of the focused tab (I.E. skills).

So, essentially, some type of update causes NVDA to not recognize the whole content, and the only way to fix the problem is to refresh NVDA's buffer using NVDA+f5.

**Comment by ehollig** [comment]
I would be interested to know if this is still the case with the latest NVDA. Unfortunately, I do not believe we can contact the original author for more information, so closing as wontfix
